zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/Follower.java

```diff
@@ -248,14 +249,9 @@ protected void processPacket(QuorumPacket qp) throws Exception {
      * @return zxid
      */
     public long getZxid() {
-        try {
-            synchronized (fzk) {
-                return fzk.getZxid();
-            }
-        } catch (NullPointerException e) {
-            LOG.warn("error getting zxid", e);
+        synchronized (fzk) {
```

Comment:
this NPE will never happens?

Reply:
Nope.  One of the changes that I made was to enforce that `fzk` be non-null:

```
this.fzk = Objects.requireNonNull(zk);
```